SpatialGDK/Documentation/content/get-started/example-project/exampleproject-local-deployment.md

```diff
@@ -50,6 +50,9 @@ To launch a local deployment:
 3. Select **Start**. This opens a terminal window and starts a local SpatialOS deployment. Your game is ready to play when you see the output `SpatialOS ready. Access the inspector at http://localhost:21000/inspector` in your terminal window.<br/>
    ![Start]({{assetRoot}}assets/screen-grabs/toolbar/start-button.png)<br/>
    _Image: On the GDK toolbar in the Editor select **Start**_</br>
+
+    <%(#Expandable title="Debugging using breakpoints?")%>Due to a [known issue](https://github.com/spatialos/UnrealGDK/issues/940) with the GDK, you need to follow [these steps]({{urlRoot}}/content/troubleshooting#q-my-worker-instances-are-being-disconnected-from-the-spatialos-runtime-unexpectedly-while-debugging-locally) before debugging your game using C++ or Blueprint breakpoints.<%(/Expandable)%>
```

Comment:
@padhansell @ElleEss I've changed the callout to an expandable for the example project steps. I think this was the only outstanding issue - please take a look and approve if you're happy!